### ISSUE TITLE
custom ad filters: cannot download

### ISSUE THREAD

**Issue by lazymonkey2** [brave/brave-browser#35444]
## Description 

I want to add custom ad filters, but it gives me error:
![image](https://github.com/brave/brave-browser/assets/26486758/de8f2055-daaf-4511-8afa-8e4172c6900e)

## Steps to Reproduce

   1.  load brave://settings/shields/filters
   2. add custom filter:  https://raw.githubusercontent.com/uBlockOrigin/uAssets/master/filters/annoyances-others.txt
   3. 

## Actual result:

error Download failed

## Reproduces how often: 

always

## Version/Channel Information:

Version 1.61.120 Chromium: 120.0.6099.234 (Official Build) (64-bit)

- Can you reproduce this issue with the current release? 
- Can you reproduce this issue with the beta channel? 
- Can you reproduce this issue with the nightly channel? 

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? 
- Does the issue resolve itself when disabling Brave Rewards? 
- Is the issue reproducible on the latest version of Chrome?

**Comment by ShivanKaul** [comment]
Works fine for me. The screenshot has a different URL than the one you posted: 
https://raw.githubusercontent.com/uBlockOrigin/uAssets/master/filters/annoyances-others.txt

**Comment by lazymonkey2** [comment]
yes, I tried multiple URLs and I copied one different from the screenshot, sorry.
Anyway it never worked correctly with this version of Brave, while using older versions there was no error.
I also created a brand new brave profile and got the same error.

**Comment by ShivanKaul** [comment]
Can you go to the URL https://raw.githubusercontent.com/uBlockOrigin/uAssets/master/filters/annoyances-others.txt normally and see the rules in the browser? Or is that blocked as well?
Would you mind taking a screenrecording showing you adding the custom filterlist and trying to subscribe?

**Comment by lazymonkey2** [comment]
still happening in v1.70.126

but I have found it only happens in one of two brave profiles.
If I add the list in a profile the it is added also in the other profile, and is working (maybe it's the default profile).
Instead if I try adding in the second profile then it is added but it's not working (in both profiles).
